src/components/ServiceList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import FilterList from "../components/FilterList";
+import useServicesList from "../hooks/useServicesList";
+import { Card, CardContent } from "@baltimorecounty/dotgov-components";
+
+const ServiceList = () => {
+  const serviceItems = useServicesList();
+
+  const serviceCard = ({ name, department, url, icon, rank }) => (
```

Comment:
Service Card should be it's own component. That makes a little simpler to a developer coming into the project and knowing where they have to go to update a card.

Reply:
done